pkg/utils/utils.go

```diff
@@ -6,38 +6,45 @@ SPDX-License-Identifier: Apache-2.0
 package utils
 
 import (
+	"maps"
 	"slices"
 	"sort"
-
-	"golang.org/x/exp/maps"
 )
 
 func Ptr[T any](t T) *T {
 	return &t
 }
 
-func SortedKeys[T ~string, V any](m map[string]map[T]V) []T {
-	keys := make([]T, 0)
-	for _, vpc := range m {
-		keys = append(keys, maps.Keys(vpc)...)
+func MapKeys[T comparable, V any](m map[T]V) []T {
+	keys := maps.Keys(m)
+	result := make([]T, 0)
+	for key := range keys {
+		result = append(result, key)
 	}
-	cmp := func(i, j int) bool { return keys[i] < keys[j] }
-	sort.Slice(keys, cmp)
+	return result
+}
 
+func SortedMapKeys[T ~string, V any](m map[T]V) []T {
+	keys := MapKeys(m)
+	slices.Sort(keys)
 	return keys
 }
 
-func SortedValuesInKey[T ~string, V any](m map[string]map[T]V, key string) []T {
-	keys := maps.Keys(m[key])
+func SortedAllInnerMapsKeys[T ~string, V any](m map[string]map[T]V) []T {
+	keys := make([]T, 0)
+	for _, vpc := range m {
+		keys = append(keys, MapKeys(vpc)...)
+	}
 	cmp := func(i, j int) bool { return keys[i] < keys[j] }
 	sort.Slice(keys, cmp)
 
 	return keys
 }
 
-func SortedMapKeys[T ~string, V any](m map[T]V) []T {
-	keys := maps.Keys(m)
-	slices.Sort(keys)
+func SortedInnerMapKeys[T ~string, V any](m map[string]map[T]V, key string) []T {
+	keys := MapKeys(m[key])
+	cmp := func(i, j int) bool { return keys[i] < keys[j] }
+	sort.Slice(keys, cmp)
```

Comment:
```suggestion
    return SortedMapKeys(m[key])
```
I'm not sure how helpful this function is, given we already have `SortedMapKeys`